media/js/src/views/quiz-validator.js

```diff
@@ -8,6 +8,38 @@ define([
      */
     var QuizValidator = Backbone.View.extend({
         /**
+         * This function returns false if all the checkboxes on the
+         * form that has class `cssClass` (default: 'quizblock-required')
+         * are unchecked. Otherwise, it returns true.
+         */
+        validateCheckboxForm: function($form, cssClass) {
+            if (typeof cssClass === 'undefined') {
+                cssClass = 'quizblock-required';
```

Comment:
This validator only looks at quizzes that have the `quizblock-required` CSS class. The CSS class is configurable.. but by default, validation is off for checkbox quizzes.